packages/components/src/badge/index.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+import { ReactNode } from 'react';
+
+const Wrapper = styled.span`
+	font-size: 12px;
+	font-weight: 500;
+	color: var( --studio-gray-80 );
+	background-color: var( --studio-gray-5 );
+	padding: 0px 10px;
+	margin-left: 10px;
+	border-radius: 4px;
+	line-height: 20px;
+`;
+
+interface BadgePrpos {
+	children: ReactNode;
+}
+
+function Badge( { children }: BadgePrpos ) {
```

Comment:
We are writing a component that has the same name as the existing [Badge Calypso UI component ](https://wpcalypso.wordpress.com/devdocs/design/badge). Will it be safe?

Reply:
@wojtekn Thanks for finding the Badge. Yesterday I couldn't find that component.
I'll rename it and probably move it to `client/components`.